ivy/func_wrapper.py

```diff
@@ -556,6 +557,26 @@ def _outputs_to_ivy_arrays(*args, **kwargs):
     return _outputs_to_ivy_arrays
 
 
+def handle_backend_invalid(fn: Callable) -> Callable:
+    @functools.wraps(fn)
+    def _handle_backend_invalid(*args, **kwargs):
+        from ivy.utils.backend.handler import _determine_backend_from_args
```

Comment:
do we need to import that lazily? This will add a bit of an overhead when calling the function.